cmd/root.go

```diff
@@ -69,16 +70,19 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&apiEndpoint, "apiendpoint", "", "Holepunch server to use - (default is https://api.holepunch.io)")
 	rootCmd.PersistentFlags().StringVar(&publicKeyPath, "publickeypath", "", "Path to your public keys - (~/.ssh)")
 	rootCmd.PersistentFlags().StringVar(&privateKeyPath, "privatekeypath", "", "Path to your private keys - (~/.ssh)")
-	rootCmd.PersistentFlags().BoolVar(&crashReporting, "crashreporting", false, "Send crash reports to the developers(Set true by default in beta)")
+	rootCmd.PersistentFlags().StringVar(&sshEndpoint, "sshendpoint", "", "endpoint that ssh tunnel connects to, to get to internal network")
```

Comment:
I still think there needs to be additional clarify between `sshEndpoint`, `baseUrl`, and `apiEndpoint`, but that could come later and maybe the more appropriate place is in the detailed help messages/docs.

Reply:
Yeah i added a card to clear up the config file

Reply:
Currently this is a pretty bad way to do it